.kokoro/nightly/integration-beam.cfg

```diff
@@ -0,0 +1,38 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Configure the docker image for kokoro-trampoline.
+env_vars: {
+  key: "TRAMPOLINE_IMAGE"
+  value: "gcr.io/cloud-devrel-kokoro-resources/java8"
+}
+
+env_vars: {
+    key: "INTEGRATION_TEST_ARGS"
+    value: "-PhbasesnapshotsIntegrationTest -Dgoogle.bigtable.project.id=gcloud-devel -Dgoogle.bigtable.instance.id=hbase-snapshot-import-test -Dgoogle.dataflow.stagingLocation=gs://java-bigtable-hbase-testing/staging -Dcloud.test.data.folder=gs://java-bigtable-hbase-testing/hbase-snapshot-import-integration-tests -Dregion=us-central1"
+}
```

Comment:
The profiles are a bit of mess:
`sequencefileIntegrationTest` will run all of the ITs but doesnt seem like it has the necessary region prop

`beamIntegrationTest` will run `CloudBigtableBeamITTest`
`hbasesnapshotsIntegrationTest` will run just the snapshots

I dont quite understand why all of these are separate profiles? I think the original reason for having a separate profile is that cloud resource names are available so we want to avoid running these during the regular verify phase.

I dont think we should be defining separate kokoro jobs for each test case. So I think all of the profiles should be merged. If the other ones are currently broken, feel free to exclude them in the failsafe config.

Reply:
Agree it make sense to let CI to run all integration tests in one profile.

However, I believe it would also be beneficial to keep these individual profiles as `hbasesnapshotsIntegrationTest` would launch multiple dataflow jobs hence it quite time consuming to run, hence if we would want to only test part of the package and get result faster, we can use those individual profiles.

I think `sequencefileIntegrationTest` is not currently running in any of our presubmit tasks, I can update the test and add the `region` argument.

Reply:
Ok. now all integration tests under `bigtable-beam-import` should be running and passing in this kokoro task.

PTAL